components/Feature/EditAction/index.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { DateInput, TextInput, Button, TextArea } from 'components/Form';
+import moment from 'moment';
+import css from './index.module.scss';
+
+const EditAction = ({ action, onActionUpdated }) => {
+  console.log({ action });
+  const year = action.dueDate.split('-')[0];
+  const month = action.dueDate.split('-')[1];
+  const day = action.dueDate.split('-')[2];
```

Comment:
We could tidy this up by [destructing the array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Examples) that gets returned, instead of manually pulling out each value.

```suggestion
  const [year, month, day] = action.dueDate.split('-');
```